src/main/java/ita/softserve/course_evaluation/dto/UserDtoMapper.java

```diff
@@ -27,8 +35,9 @@ public static User fromDto(UserDto dto) {
 		user.setFirstName(dto.getFirstName());
 		user.setLastName(dto.getLastName());
 		user.setEmail(dto.getEmail());
-		user.setPassword(dto.getPassword());
+		user.setPassword(passwordEncoder.encode(dto.getPassword()));
```

Comment:
This was already happening in the AuthController during registration

Reply:
it is for Admin mode meybe usefull